Formula/redo.rb

```diff
@@ -0,0 +1,38 @@
+class Redo < Formula
+  include Language::Python::Virtualenv
+
+  desc "Implements djb's redo: an alternative to make"
+  homepage "https://redo.rtfd.io/"
+  url "https://github.com/apenwarr/redo/archive/redo-0.42a.tar.gz"
+  sha256 "0ff0fd29f2767fe33374cfac97fd13b3dd27207bdacbeb5e02039b992d4fb815"
+
+  depends_on "python"
```

Comment:
Could you make it depend on python@3.8?

Reply:
This should be a very easy fix. Mind if I test this last and then commit all the changes at once?